src/components/common/Stepper.vue

```diff
@@ -177,14 +182,58 @@ export default class Stepper extends Vue {
 @import '@/assets/styles/theme.scss';
 
 #step-buttons-container {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 0;
+  padding: 2rem 0;
   background: $BCgovInputBG;
 }
 
-.mdi-check-circle {
+.step {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  align-items: center;
+  justify-content: center;
+}
+
+.step__indicator {
+  position: relative;
+  width: 100%;
+  text-align: center;
+}
+
+.step__line {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  height: 2px;
+  width: 100%;
+  background-color: $gray5;
+  transform: translateX(-50%);
+}
+
+.step:first-child .step__line {
+  display: none;
+}
+
+.step__icon {
+  position: relative;
+  background-color: $BCgovInputBG;
+  z-index: 2;
+}
+
+.step__icon2 {
   position: relative;
   margin-top: -32px;
   margin-left: -16px;
   background: $BCgovInputBG;
   border-radius: 50%;
+  z-index: 3;
+}
+
+.step__label {
+  margin-top: 10px;
```

Comment:
Same q as above!

Reply:
I guess I didn't have a good reason to use rems. This just seemed like a good value. It's also easier to read than 0.625rem.
To keep the number of font sizes to a minimum I use a small set of rem values for fonts. And I use a small set of values for margins and padding. But here I'm just tweaking the space between elements so px is more specific (and there's no need to scale it).

Reply:
Sounds good, was just curious!